renovate.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "reviewers": ["welpaolo","deusebio","marcoppenheimer"],
+  "enabledManagers": ["poetry", "pip_requirements", "github-actions", "regex"],
+  "schedule": ["after 1am and before 2am every weekday"],
```

Comment:
**todo** I would make it once per week, running saturday or sunday to review dependencies PR on mondays